examples/basics/plotting/quick.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+Example of plotting multiple things using the quick interface.
+"""
+
+import numpy as np
+
+from vispy.io import load_data_file
+import vispy.plot.q as vpq
```

Comment:
Well, if users have to adjust to an API, then I think it makes sense to just adopt the other one in this PR, and scrap the `quick` one entirely. In other words, just have this one:
```
fig = Fig()
fig[0, 0].plot(...)
```
For now at least I'll remove the `quick` stuff, and we can figure out later if we really need an additional API or not.